swf/src/read.rs

```diff
@@ -52,6 +52,33 @@ pub fn extract_swz(input: &[u8]) -> Result<Vec<u8>> {
     Err(Error::invalid_data("Invalid ASN1 blob"))
 }
 
+/// Parses an SWF from data in an SWF projector bundle
+/// Returns an `Error` if this is not a valid SWF projector bundle.
+pub fn decompress_projector_bundle(data: &[u8]) -> Result<SwfBuf> {
```

Comment:
I wouldn't call this `decompress_...`, as there is only a bit of slicing done.
Maybe `extract_swf_from_...` - but something shorter? :P

Reply:
Oh the  return type changed. The latest commit gives the same return value as "decompress_swf".

The idea was since this is a fallback, you first try decompress_swf, and if that fails, you try to get data via decompress_projector_bundle